src/main/sensors/voltage.c

```diff
@@ -257,6 +285,8 @@ void voltageMeterESCReadCombined(voltageMeter_t *voltageMeter)
 #else
     voltageMeter->filtered = voltageMeterESCState.voltageFiltered;
     voltageMeter->unfiltered = voltageMeterESCState.voltageUnfiltered;
+    if (systemConfig()->vbatSagCompEnabled)
+        voltageMeter->sagFiltered = voltageMeterESCState.voltageSagFiltered;
```

Comment:
Why are you doing this? You are already setting `voltageMeter->sagFiltered` in https://github.com/betaflight/betaflight/pull/9561/files#diff-0ba5d14e9202d64089d5c070d3745795R196 - what you are doing here will turn sag filtering into a complete mess if it is ever enabled with ESC sensor feedback.